src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -372,6 +436,67 @@ g.test('formats')
     t.expectOK(result);
   });
 
+g.test('default_blend_color,initial_blend_color')
+  .desc(`Test that the initial blend color is correctly set at the beginning of every sub passes.`)
+  .fn(async t => {
+    const kFormat = 'rgba8unorm';
+    const kSize = 64;
+    const kWhiteColorData = new Float32Array([255, 255, 255, 255]);
+    const kBlackColorData = new Float32Array([0, 0, 0, 0]);
+
+    const basePipeline = t.createRenderPipelineForTest(kFormat, undefined);
+    const testPipeline = t.createRenderPipelineForTest(kFormat, {
+      srcFactor: 'constant',
+      dstFactor: 'one',
+      operation: 'add',
+    });
+
+    const renderTarget = t.device.createTexture({
+      usage: GPUTextureUsage.RENDER_ATTACHMENT | GPUTextureUsage.COPY_SRC,
+      size: [kSize, kSize],
+      format: kFormat,
+    });
+
+    const commandEncoder = t.device.createCommandEncoder();
+    const renderPass = commandEncoder.beginRenderPass({
+      colorAttachments: [
+        {
+          view: renderTarget.createView(),
+          loadOp: 'load',
+          storeOp: 'store',
+        },
+      ],
+    });
+    renderPass.setPipeline(basePipeline);
+    renderPass.setBindGroup(
+      0,
+      t.createBindGroupForTest(basePipeline.getBindGroupLayout(0), kBlackColorData)
+    );
+    renderPass.draw(3);
```

Comment:
I don't think this draw call has any effect. The texture should already be filled with zeroes when it's created, and I think this just re-fills it with zeroes. It's possible that, when the Dawn test was written, we needed this because we didn't have proper resource zeroing yet, but we don't need that here, so let's remove this draw call.

Reply:
Thank you for pointing it out. I called the draw referring to the end2end's test.  Ok, let me remove the call both.